### ISSUE TITLE
offensive examples are unnecessary

### ISSUE THREAD

**Issue by mjhaller** [YOURLS/YOURLS#2459]
I'd like to try to make a gentle suggestion about this:

https://github.com/YOURLS/YOURLS/blob/master/user/config-sample.php#L88

There is no real reason to provide these particular examples, which are like a police lineup of some of the most offensive english words known.  Practically no english readers want to see these words, which alone seem worse than if they were combined with the 140 other words that generally make such lists.  

What I am saying is that I don't think you want these particular ones as representatives in your source code.  If users of this codebase wish to implement a ban list, there are several available online.

**Comment by ozh** [comment]
They are examples. For instance "sex" is a pretty easy combination to obtain if you use YOURLS with all defaults on and generate short URLs sequentially.

I'm totally ok with these words being in the source code. I think the example is a bit too wordy indeed (no need for so many words, I concede that) but removing some of them after all these years would feel like an unwanted victory of "politically correct" over "spontaneous and legacy". Who fucking cares about a few fucking swears in the code? Not fucking me.

**Comment by ozh** [comment]
(no offense intended, of fucking course)

**Comment by mjhaller** [comment]
Those aren't just swears.  I like swears in code, but they are also offensive slurs.

Personally, I wouldn't see this as a victory of political correctness.  I would see it as a simple acceptance of a grand suggestion.  :)
